tests/integration/gov/keeper/tally_test.go

```diff
@@ -89,9 +89,10 @@ func TestTallyOnlyValidatorsAllYes(t *testing.T) {
 	assert.Assert(t, ok)
 	passes, burnDeposits, tallyResults, _ := f.govKeeper.Tally(ctx, proposal)
 
-	assert.Assert(t, passes)
+	// HV2: weighted operations not allowed
```

Comment:
Same like, it doesnt seems like test for weighted vote. The regular vote function uses the same type - https://github.com/0xPolygon/cosmos-sdk/blob/main/x/gov/keeper/msg_server.go#L223-L233

Reply:
Same as before, changed the comment. Thanks.